packages/components/src/templates/next/components/internal/ScrollToTop.tsx

```diff
@@ -0,0 +1,22 @@
+"use client"
+
+import { useEffect } from "react"
+import { usePathname } from "next/navigation"
```

Comment:
not a review but saw this, `next/navigation` is not available in components so we will need to depend on `window` if we need the path name